File: 33-file-cat/file_cat.pl
# Perl program to print file contents
# Written by M.V.Harish Kumar on 14-01-2025
open FILE, '<', $ARGV[0] or die "Can't open file: $!";
print "Contents of $ARGV[0]:\n";
print '-'x20 . "\n";
print <FILE>;
print "\n". '-'x20 . "\n";

File: 33-file-cat/file_cat.exs
# Elixir program to print file contents
# Written by M.V.Harish Kumar on 06-02-2025
fname = System.argv |> Enum.fetch!(0)
IO.puts "Contents of #{fname}"
"-" |> String.duplicate(20) |> IO.puts
File.read!(fname) |> IO.puts
"-" |> String.duplicate(20) |> IO.puts
